src/main/java/thePackmaster/cardmodifiers/basicspack/DuplicateModifier.java

```diff
@@ -0,0 +1,59 @@
+package thePackmaster.cardmodifiers.basicspack;
+
+import basemod.abstracts.AbstractCardModifier;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cardmodifiers.madsciencepack.AbstractMadScienceModifier;
+
+public class DuplicateModifier extends AbstractMadScienceModifier {
+
+    public DuplicateModifier(int valueIn){
+        super(valueIn);
+    }
+
+    @Override
+    public String modifyDescription(String rawDescription, AbstractCard card) {
+        if (this.value == 1) {
+            return rawDescription + CardCrawlGame.languagePack.getUIString(SpireAnniversary5Mod.makeID("DuplicateModifier")).TEXT[0];
```

Comment:
Save the UI string in a static variable instead of retrieving it every frame for every card it's on